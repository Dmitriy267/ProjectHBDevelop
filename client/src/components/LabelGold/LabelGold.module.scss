.label__div{
    background: linear-gradient(106.06deg, #FFD54F 1.12%, #FEC100 97.96%);
    border-radius: 0px 10px 10px 10px;
    width: 66px;
    height: 37px;
    z-index: 1;
    display: flex;
    align-items: center;
    justify-content: center;
    bottom: 238px;
    left: 422px;
    position: absolute;
    top:0px;
    left:340px;
}



.label__div p{
    width: 42px;
height: 22px;
font-family: 'Inter Bold';
font-size: 18px;
line-height: 22px;
color:#fff;
text-shadow: 0px 1px 0px rgba(147, 79, 0, 0.3);
}