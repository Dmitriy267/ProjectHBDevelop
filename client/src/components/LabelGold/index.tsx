import React from "react";
import styles from './LabelGold.module.scss';
function LabelGold (){
    return(
        <div className={styles.label__div}>
            <p>Gold</p>
        </div>
    )
}
export default LabelGold