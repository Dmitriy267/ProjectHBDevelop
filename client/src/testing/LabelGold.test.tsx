import React from "react";
import { render, screen } from "@testing-library/react";
import renderer from 'react-test-renderer';
import LabelGold from "../components/LabelGold";

  describe ('LabelCold component', ()=>{
    test('The correctli snapshot component', ()=>{
        const component=renderer.create(<LabelGold/>);
        const tree=component.toJSON();
        expect(tree).toMatchSnapshot();
    });
    test('it renders', ()=>{
        render(<LabelGold/>);
        expect(screen.getByText('Gold')).toBeInTheDocument()
    })
  })
   