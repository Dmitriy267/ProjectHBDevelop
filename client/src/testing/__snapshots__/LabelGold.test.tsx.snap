// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelCold component The correctli snapshot component 1`] = `
<div
  className="label__div"
>
  <p>
    Gold
  </p>
</div>
`;
